XF/Repository/UserAlert.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace SV\UserMentionsImprovements\XF\Repository;
+
+class UserAlert extends XFCP_UserAlert
+{
+	public function alert(
+		\XF\Entity\User $receiver, $senderId, $senderName,
+		$contentType, $contentId, $action, array $extra = []
+	)
+	{
+		if ($this->app()
+				->offsetExists('sv_userGroupMentionedIds') && ($contentType == 'post' || $contentType == 'profile_post') && $action == 'mention' && array_key_exists($receiver->user_id, $this->app()->sv_userGroupMentionedIds)
```

Comment:
I don't think it make sense for the explicit content type check, as without it; it allows easily extending to other content types (ie reports) without needing to edit to extend this code.